pkg/utils/remotesubnet.go

```diff
@@ -0,0 +1,13 @@
+package utils
+
+import (
+	"fmt"
+)
+
+const remoteSubnetNameFormat = "%v.%v"
```

Comment:
I think it is not bad because subnet is already separated by "-", using another separator can make the name more precise. For example, `"remotecluster-2.overlay-subnet1"` vs `"remotecluster-2-overlay-subnet1"`

Reply:
Got it.